e2e test "TestSimpleIngress" is flaky

2 runs failed out of 4 attempts for the same `TestSimpleIngress` case in `//tests/e2e/tests/simple:go_default_test`. 

Seemingly related logs:
```
W1215 02:10:39.283] I1215 02:10:39.282370     891 framework.go:200] Running test
W1215 02:10:39.284] I1215 02:10:39.282415     891 simple1_test.go:71] Fetching 'http://104.197.124.154/fortio/debug'
W1215 02:10:39.318] W1215 02:10:39.317991     891 simple1_test.go:79] Attempt 1 : http.Get error Get http://104.197.124.154/fortio/debug: dial tcp 104.197.124.154:80: getsockopt: connection refused
I1215 02:10:39.419] === RUN   TestSimpleIngress
W1215 02:10:49.355] W1215 02:10:49.354278     891 simple1_test.go:79] Attempt 2 : http.Get error Get http://104.197.124.154/fortio/debug: dial tcp 104.197.124.154:80: getsockopt: connection refused
W1215 02:10:59.391] W1215 02:10:59.390081     891 simple1_test.go:79] Attempt 3 : http.Get error Get http://104.197.124.154/fortio/debug: dial tcp 104.197.124.154:80: getsockopt: connection refused
W1215 02:11:09.427] W1215 02:11:09.425771     891 simple1_test.go:79] Attempt 4 : http.Get error Get http://104.197.124.154/fortio/debug: dial tcp 104.197.124.154:80: getsockopt: connection refused
W1215 02:11:19.462] W1215 02:11:19.461561     891 simple1_test.go:79] Attempt 5 : http.Get error Get http://104.197.124.154/fortio/debug: dial tcp 104.197.124.154:80: getsockopt: connection refused
W1215 02:11:29.499] W1215 02:11:29.497849     891 simple1_test.go:79] Attempt 6 : http.Get error Get http://104.197.124.154/fortio/debug: dial tcp 104.197.124.154:80: getsockopt: connection refused
I1215 02:11:39.535] --- FAIL: TestSimpleIngress (60.25s)
I1215 02:11:39.535] 	simple1_test.go:98: Unable to find expected output after 7 attempts
```
(from https://k8s-gubernator.appspot.com/build/istio-prow/e2e-cluster_wide-auth/228/)

So simply timeout. Maybe increase the number of max attempts?

--- a/tests/e2e/framework/testInfo.go
+++ b/tests/e2e/framework/testInfo.go
@@ -139,6 +139,10 @@ func (t testInfo) FetchAndSaveClusterLogs(namespace string) error {
 		return nil
 	}
 
+	_, err := util.Shell("kubectl get ingress --all-namespaces")
+	if err != nil {
+		return err
+	}
 	lines, err := util.Shell("kubectl get pods -n " + namespace)
 	if err != nil {
 		return err

--- a/tests/e2e/tests/simple/simple1_test.go
+++ b/tests/e2e/tests/simple/simple1_test.go
@@ -71,7 +71,7 @@ func TestSimpleIngress(t *testing.T) {
 	// with "echo debug server ..." on the /debug uri.
 	url := "http://" + tc.Kube.Ingress + "/fortio/debug"
 	log.Infof("Fetching '%s'", url)
-	attempts := 7 // should not take more than 70s to be live...
+	attempts := 1 // should not take more than 70s to be live...
 	for i := 1; i <= attempts; i++ {
 		if i > 1 {
 			time.Sleep(10 * time.Second) // wait between retries
@@ -97,7 +97,7 @@ func TestSimpleIngress(t *testing.T) {
 		}
 		return // success
 	}
-	t.Errorf("Unable to find expected output after %d attempts", attempts)
+	t.Errorf("Unable to find expected output after %d attempts - ingress issue", attempts)
 }
 
 func TestSvc2Svc(t *testing.T) {

--- a/tests/e2e/tests/simple/simple1_test.go
+++ b/tests/e2e/tests/simple/simple1_test.go
@@ -66,15 +66,17 @@ func TestMain(m *testing.M) {
 	os.Exit(tc.RunTest(m))
 }
 
+// TODO: need an "is cluster ready" before running any tests
+
 func TestSimpleIngress(t *testing.T) {
 	// Tests the rewrite/dropping of the /fortio/ prefix as fortio only replies
 	// with "echo debug server ..." on the /debug uri.
 	url := "http://" + tc.Kube.Ingress + "/fortio/debug"
 	log.Infof("Fetching '%s'", url)
-	attempts := 1 // should not take more than 70s to be live...
+	attempts := 7 // should not take more than 1min45s to be live...
 	for i := 1; i <= attempts; i++ {
 		if i > 1 {
-			time.Sleep(10 * time.Second) // wait between retries
+			time.Sleep(15 * time.Second) // wait between retries
 		}
 		resp, err := http.Get(url)
 		if err != nil {

